vllm/entrypoints/openai/serving_engine.py

```diff
@@ -74,8 +75,8 @@ def __init__(
         self.prompt_adapter_requests = []
         if prompt_adapters is not None:
             for i, prompt_adapter in enumerate(prompt_adapters, start=1):
-                with open(f"./{prompt_adapter.local_path}"
-                          f"/adapter_config.json") as f:
+                with open(pathlib.Path(
+                        prompt_adapter.local_path).resolve()) as f:
```

Comment:
```suggestion
                with pathlib.Path(prompt_adapter.local_path).open() as f:
```

Reply:
Also, is it intended that `adapter_config.json` is removed?

Reply:
Nope. Thanks for the catch and suggestion.